backend/app/api/v1/commons/example_responses.py

```diff
@@ -119,12 +119,35 @@ def response_422():
     ]
 }
 
+telco_response_example ={
+    "startDate": "2023-09-20",
+    "endDate": "2023-09-20",
+    "results": [
+        {
+            "ciSystem": "Jenkins",
+            "uuid": "2cc5d4ca895ca5d84cab0fd7923db93b",
+            "encrypted": "gAAAAABmQALtP0g5UPMsOjQw46tZ-aBz77yl-8QNI4jwLfIEV1POnOlA1ny89cp3Nrik3OzpNwXrWO3K4ZwtOliTfk0SO5NkNZHY8reJhvOVJBGFEw2enyjRaHp9hIaJdE0Vrfuqt_NjiYX-vOZo0Sjc84R76LvxjAC6f_urceGGZICH36IkT2g=",
+            "releaseStream": "Release Candidate",
+            "jobStatus": "success",
+            "buildUrl": "https://ci-jenkins-csb-kniqe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-far-edge-vran-tests/532",
```

Comment:
obfuscate the URL

Reply:
Updated as suggested. Thanks for catching this.